config/settings.yml

```diff
@@ -1145,6 +1145,7 @@ veteran_readiness_and_employment:
       - Jason.Wolf@va.gov
       - Kathleen.Crawford@va.gov
       - Kevin.Musiorski@va.gov
+      - Amanda.Leaders@va.gov
     staging_emails:
```

Comment:
One thought (or two), take it or leave it...

You can also probably take out this `staging_emails:` key and the single element array now that we don't use it. Or keep it and maybe only comment out the other line:

```
# Settings.veteran_readiness_and_employment.daily_report.staging_emails.dup
return
```
...in case we want to turn it on one day.

Reply:
I think I'm going to leave the array there (just in case it has downstream effects I didn't see) but I like the idea of leaving that line but commenting it out, in case we do want to come back to it later.